### PR TITLE
Update legacy_plugins.md to include HBM authz plugin

### PR DESCRIPTION
**Pull request by ghost**
- What I did
Update legacy_plugins.md to include HBM authz plugin

### CODE REVIEW

File: docs/extend/legacy_plugins.md [hunk 1]
```diff
@@ -88,6 +88,7 @@ Plugin
  Plugin                                                       | Description
 ------------------------------------------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
 [Casbin AuthZ Plugin](https://github.com/casbin/casbin-authz-plugin) | An authorization plugin based on [Casbin](https://github.com/casbin/casbin), which supports access control models like ACL, RBAC, ABAC. The access control model can be customized. The policy can be persisted into file or DB.
+[HBM plugin](https://github.com/kassisol/hbm)                | An authorization plugin.
```
Comment (thaJeztah) [inline docs/extend/legacy_plugins.md]: Could you add a bit more information to the description? I see there's some information about the plugin in the readme; https://github.com/kassisol/hbm (also mentions that `HBM` stands for "Harbormaster", etc)

**Review by thaJeztah — CHANGES_REQUESTED** [review]

File: docs/extend/legacy_plugins.md (hunk 1 above)
Comment (juliengk) [inline docs/extend/legacy_plugins.md]: I have updated the change with your recommendations. Thanks

**Review by boaz0 — APPROVED** [review]
👍

**Comment by cpuguy83** [comment]
Thanks for adding!
Unfortunately these docs have moved to github.com/docker/cli since you opened this.
Can you open a new PR on that repo?

Closing this one since we can't merge it here.
Thanks!

**Pull request closed without merging**